build/manifest.ps1

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+        Provides the list of artifacts (Packages and Assemblies) generated by this repository.
+    
+    .PARAMETER OutputFormat
+        Specifies if the output of this script should be a hashtable with the artificats
+        as strings with the absolute path (AbsolutePath) or FileInfo structures.
+#>
+param(
+    [ValidateSet('FileInfo','AbsolutePath')]
+    [string] $OutputFormat = 'FileInfo'
+);
+
+& "$PSScriptRoot/set-env.ps1"
+
+$artifacts = @{
+    Packages = @(
+        "Microsoft.Quantum.Compiler"
```

Comment:
Once #697 is merged, I believe `"Microsoft.Quantum.DocumentationGenerator"` also needs to be added here. @cgranade to confirm.

Reply:
Yes, that will need to be added to the list of packages shortly.